src/NuGet.Clients/NuGet.PackageManagement.UI/UserInterfaceService/NuGetUI.cs

```diff
@@ -315,6 +315,10 @@ public void EndOperation()
 
         public (string modelVersion, string vsixVersion)? RecommenderVersion { get; set; }
 
+        public int TopLevelVulnerablePackagesCount { get; set; }
+
+        public int TopLevelVulnerablePackagesMaxSeverity { get; set; }
```

Comment:
Do you know why `int` is being used to represent severity? 
According to https://docs.microsoft.com/en-us/nuget/api/registration-base-url-resource#vulnerabilities, there are only 4 values. I'd expect an `enum` or similar finite construct.

Reply:
Because I'm borrowing Severity definition from:

https://github.com/NuGet/NuGet.Client/blob/4ba39b98697e715fd09ce762f59c27dc0809eea6/src/NuGet.Clients/NuGet.VisualStudio.Internal.Contracts/ContextInfos/PackageVulnerabilityMetadataContextInfo.cs#L17

Also, it's easier to compare vulnerability severities using int's

Reply:
`enum`s are also easy to compare... see my other comment

Reply:
Other thread has a follow-up issue for refactoring int to enum.